bin/reth/src/dirs.rs

```diff
@@ -12,8 +12,8 @@ use std::{
 /// Constructs a string to be used as a path for configuration and db paths.
 pub fn config_path_prefix(chain: Chain) -> String {
     match chain {
```

Comment:
here whether I use Named of from_named, I am getting the same errror, need some help here?

Reply:
I think you can use 
```
match chain {
        name => name.to_string(),
        id => id.to_string(),
    }

```

Reply:
I think this can be used

```
pub fn config_path_prefix(chain: Chain) -> String {
    if let Some(named) = chain.named() {
        named.to_string()
    } else {
        chain.id().to_string()
    }
}

```

Reply:
should we move this to a `Display` / `ToString` impl?

Reply:
seems like `Chain` already implements `Display` (which gets us `ToString`)